modules/gapi/perf/common/gapi_render_perf_tests.cpp

```diff
@@ -0,0 +1,1257 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#include "../perf_precomp.hpp"
+#include "../../src/api/render_priv.hpp"
+#include "../../test/common/gapi_tests_common.hpp"
+#include <opencv2/gapi/imgproc.hpp>
+
+#ifdef HAVE_FREETYPE
+#include <codecvt>
+#endif // HAVE_FREETYPE
+
+namespace opencv_test
+{
+  using namespace perf;
+
+  #ifdef HAVE_FREETYPE
+  class RenderTestFTexts : public TestPerfParams<tuple<std::wstring, cv::Size, cv::Point,
```

Comment:
Remove #ifdef's

Reply:
Do same thing for unit tests

Reply:
There isn't changes for unit tests. But ok, let's do it in separate PR